src/assets/Generator.Shared/OperationInternalsOfT.cs

```diff
@@ -21,12 +21,12 @@ namespace Azure.Core
     /// </summary>
     /// <typeparam name="T">The final result of the LRO.</typeparam>
 #pragma warning disable SA1649 // File name should match first type name
-    internal class OperationInternals<T> : OperationInternals
+    internal class OperationInternals<T>: IOperation<T>
 #pragma warning restore SA1649 // File name should match first type name
     {
         private readonly IOperationSource<T> _source;
```

Comment:
In the end of the series of refactorings are we still going to keep the `IOperationSource<T>`?

Reply:
No, will be removed as well.